tests/tests.lua

```diff
@@ -186,7 +169,7 @@ describe("tests on Lua bounty", function()
                                 ImgLink = "",
                                 Name = "Developer1",
                             },
-                            Value = tohex(1000),
+                            Value = "1000",
```

Comment:
It seems that values are encoded as decimal strings rather than hexadecimal strings.